cross-chain/base/deploy_l2/14_update_with_polygon_in_wormhole_gateway_mapping.ts

```diff
@@ -11,7 +11,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     "0x1af5DC16568EFF2d480a43A77E6C409e497FcFb9"
 
   // See https://book.wormhole.com/reference/contracts.html
-  // This ID is valid for both Polygon Testnet (Mumbai) and Mainnet
+  // This ID is valid for both Polygonn Goerli-based Testnet (Mumbai) and
+  // Mainnet. Wormhole does not support the Sepolia-based Amoy Testnet yet.
+  // TODO: Update the ID once the support is added.
   const polygonWormholeChainID = 5
```

Comment:
Not sure how the deploy on `baseSepolia` will behave when the `polygonWormholeChainID` does point to the value which is good for Mumbai and Polygon Mainnet, but not for Sepolia--based Amoy... Should I skip this script for now?

Reply:
Until new Polygon testnet is supported by Wormhole, we can skip this script during deploy.